Development/nmos/settings.cpp

```diff
@@ -151,7 +158,12 @@ namespace nmos
         {
             hosts.push_back(get_host_name(settings));
         }
-        if (0 != (mode & nmos::experimental::href_mode_addresses))
+        const auto server_address = experimental::get_server_address(settings);
```

Comment:
Not sure this is necessary. If the app only wants to advertise one address, then can't that be `host_address`?

Reply:
Yes, we can use the href_mode or the host_address to advertise for one address.